Assets/MixedRealityToolkit-Tests/Core/TestFixture_01_MixedRealityToolkitTests.cs

```diff
@@ -0,0 +1,543 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Interfaces;
+using Microsoft.MixedReality.Toolkit.Core.Services;
+using Microsoft.MixedReality.Toolkit.Tests.Services;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests.Core
+{
+    public class TestFixture_01_MixedRealityToolkitTests
+    {
+        #region Service Locator Tests
+
+        [Test]
+        public void Test_01_InitializeMixedRealityToolkit()
+        {
+            TestUtilities.CleanupScene();
+
+            TestUtilities.InitializeMixedRealityToolkit();
+            GameObject gameObject = GameObject.Find(nameof(MixedRealityToolkit));
+            Assert.AreEqual(nameof(MixedRealityToolkit), gameObject.name);
+        }
+
+        [Test]
+        public void Test_02_TestNoMixedRealityConfigurationFound()
+        {
+            LogAssert.Expect(LogType.Error, "No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit");
+
+            MixedRealityToolkit.Instance.ActiveProfile = null;
```

Comment:
This test appears to set the value it is expecting and only truly checking the log string... Is this intended?

Since the log message precedes setting ActiveProfile to null, it appears that

MixedRealityToolkit.Instance.ActiveProfile = null;

is not necessary for the test.

Reply:
The test is that it fails, so yes.

That's what was there previously. This test was not modified.

Reply:
> That's what was there previously. This test was not modified.

Understood. I happened to notice it on this review. Sorry.

> The test is that it fails, so yes.

Is the test that you explicity set ActiveProfile to null and then expect the values to be updated? Or is it that not initializing has the values set as asserted?

Given that the log assert is above setting the profile to null, i suspect the latter.

Reply:
Yes the latter, but the first sounds like a nice addition to our tests.